src/tools/toolstate.toml

```diff
@@ -26,7 +26,7 @@
 miri = "Broken"
 
 # ping @Manishearth @llogiq @mcarton @oli-obk
-clippy = "Broken"
+clippy = "Compiling"
```

Comment:
Is it intended it is changed to `Compiling` not `Testing`?

Reply:
Yea, testing wasn't allowed until a few hours ago: https://github.com/nrc/dev-tools-team/issues/18#issuecomment-343050790

I'll rebase to fix the failure and enable testing